tests/unit/src/test/scala/tests/OverridesSuite.scala

```diff
@@ -0,0 +1,157 @@
+package tests
+
+import com.sourcegraph.lsif_semanticdb.Symtab
+
+import scala.meta.Input
+import munit.{FunSuite, TestOptions}
+
+class OverridesSuite extends FunSuite with TempDirectories {
+
+  val targetroot = new DirectoryFixture()
+
+  override def munitFixtures: Seq[Fixture[_]] =
+    super.munitFixtures ++ List(targetroot)
+
+  def checkOverrides(
+      options: TestOptions,
+      source: String,
+      extractSymbol: String,
+      fn: (Int, Array[String]) => Unit,
+      qualifiedClassName: String = "example.Parent"
+  ): Unit = {
+    test(options) {
+      val compiler = new TestCompiler(targetroot())
+      val relativePath = qualifiedClassName.replace('.', '/') + ".java"
+      val input = Input.VirtualFile(relativePath, source)
+      val result = compiler.compileSemanticdb(List(input))
+      val symtab = new Symtab(result.textDocument)
+
+      val count = symtab.symbols.get(extractSymbol).getOverriddenSymbolsCount
```

Comment:
Another way to write these tests is like this
```suggestion

import scala.jdk.CollectionConverters._
def checkOverrides(
      options: TestOptions,
      source: String,
      extractSymbol: String,
      expected: String
      
     // ...
  val obtained = symtab.symbols.get(extractSymbol).getOverriddenSymbolsList.asScala.mkString("\n")
  assertNoDiff(obtained, expected)
}


checkOverrides(
    "same file",
    """package example;
      |
      |class Parent {
      | public void stuff() {}
      |
      | class Child extends Parent {
      |   @Override
      |   public void stuff() {}
      | }
      |}
      |""".stripMargin,
    "example/Parent#Child#stuff()."
    "example/Parent#stuff()."
  )
   
```

This avoids the repetitive `assertEquals(count, 1) ...`  body for every test case. You can also copy-paste the `""".stripMargin` formatted error message to update the expected value.